### DISCUSSION TITLE
/MAT/LAW69 how to output fitted curve

### DISCUSSION THREAD

**Discussion by aru2244** [OpenRadioss/OpenRadioss#916] [Q&A] [answered]
Hi
I have tried using /MAT/LAW69 Ogden Formulation with npair 3 . I want to print the fitted curve output . 
In LSDyna it is possible to output it. Please help

**Comment by MarianBulla** [comment, marked as answer]
Hi,
please review the starter output file (*_0000.out).
Hope it helps you ?
Thanks and best regards,
Marian

**Reply by aru2244** [reply]
Thank You
